docs/iris/src/whatsnew/latest.rst

```diff
@@ -10,8 +10,14 @@ This document explains the changes made to Iris for this release
    :depth: 3
 
 
-Features
-========
+🎉 Announcements
+================
+
+* N/A
+
+
+⚙ Features
```

Comment:
```suggestion
✨ Features
```
The current cog/gear is rendering as a compass symbol in some places. (I appreciate any change to this one might also require a change to [ISSUE_TEMPLATE/feature-request.md](https://github.com/SciTools/iris/blob/6b675e34d9aef7b29d84ce9911d76f46a2d46230/.github/ISSUE_TEMPLATE/feature-request.md))